README.md

```diff
@@ -97,6 +97,25 @@ pip install 'git+git://github.com/Contrast-Security-OSS/vulnpy#egg=vulnpy[wsgi]'
 can be used with a variety of frameworks. For example, Pylons provides a `Cascade` class,
 which can be used to compose WSGI applications serially.
 
+### Bottle
+
+Install vulnpy with bottle extensions:
+
+```
+pip install 'git+git://github.com/Contrast-Security-OSS/vulnpy#egg=vulnpy[bottle]'
+```
+
+Use the `add_vulnerable_routes` function to register vulnpy's routes with your `Falcon.API`
+application object:
+
+```py
+from bottle import Bottle
+from vulnpy.bottle.vulnerable_routes import add_vulnerable_routes
```

Comment:
could you add an import alias for `add_vulnerable_routes` so that it's available as `vulnpy.bottle.add_vulnerable_routes`? This would keep the API consistent with falcon. You can see the example in `vulnpy/falcon/__init__.py`

Reply:
If you're talking about this "from .vulnerable_routes import add_vulnerable_routes  # noqa: F401" it is already present in /bottle/__init__.py

Reply:
🤦  totally missed it

Can you then update the all import statements in the README, tests, and sample app to use
```
from vulnpy.bottle import add_vulnerable_routes
```

sorry for being nitpicky here but this is the official public API we're exposing from vulnpy so I want to make sure it's as clean as possible